cmd/virt-launcher/node-labeller/node-labeller.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
```

Comment:
should we have `-o pipefail` here as well?

Reply:
make sense, updated

Reply:
pipefail breaks non kvm environments like github actions where emulation should be used.

Reply:
@qinqon I still think that pipefil is the right thing to do.

We should probably properly detect the case where kvm is not present (instead of just continuing).